src/lib/services/identity.ts

```diff
@@ -23,7 +23,7 @@ const getMemberBySelf = async () => {
 
   if (res.ok) {
     const member = await res.json()
-    return member.address
+    return member
```

Comment:
I genuinely believe less is more, but this is also could be the result of eslint. 
`if (res.of) return await res.json()` - feel free to ignore.

Reply:
will change